components/cards/ParcelInfo.tsx

```diff
@@ -276,37 +279,112 @@ function ParcelInfo(props: ParcelInfoProps) {
       <AuctionInstructions />
     </>
   );
+  // <h1 style={{ fontSize: "1.5rem", fontWeight: 600 }}>
+  //   {account.toLowerCase() == licenseOwner?.toLowerCase()
+  //     ? "Reclaim"
+  //     : "Foreclosure Claim"}
+  // </h1>
 
-  let title;
+  let header;
   if (
     interactionState == STATE.CLAIM_SELECTING ||
     interactionState == STATE.CLAIM_SELECTED
   ) {
-    title = (
+    header = (
       <>
-        <h1 style={{ fontSize: "1.5rem", fontWeight: 600 }}>Claim a Parcel</h1>
-      </>
-    );
-  } else if (interactionState == STATE.PARCEL_RECLAIMING) {
-    title = (
-      <>
-        <h1 style={{ fontSize: "1.5rem", fontWeight: 600 }}>
-          {account.toLowerCase() == licenseOwner?.toLowerCase()
-            ? "Reclaim"
-            : "Foreclosure Claim"}
-        </h1>
+        <Row className="mb-3">
+          <Col sm="10">
+            <h1 style={{ fontSize: "1.5rem", fontWeight: 600 }}>
+              Claim a Parcel
+            </h1>
+          </Col>
+          <Col sm="2">
+            <div className="text-end">
+              <Button
+                variant="link"
+                size="sm"
+                onClick={() => setInteractionState(STATE.VIEWING)}
+              >
+                <Image src="close.svg" />
+              </Button>
+            </div>
+          </Col>
+        </Row>
       </>
     );
   } else {
-    title = (
+    const spatialURL = `https://geoweb.app?longitude=${selectedParcelCoords.x}&latitude=${selectedParcelCoords.y}`;
```

Comment:
Could we pull out the domain into a configurable constant in `lib/constants.ts`?